src/nifreeze/estimator.py

```diff
@@ -20,7 +20,7 @@
 #
 #     https://www.nipreps.org/community/licensing/
 #
-"""A model-based algorithm for the realignment of dMRI data."""
+"""A model-based algorithm for head motion and eddy current distortion estimation and realignment for 4D neuroimaging data."""
```

Comment:
Description should be a one-liner, what about this?

```suggestion
"""Orchestrates model and registration in volume-to-volume artifact estimation."""
```